tests/test_process.py

```diff
@@ -912,7 +912,7 @@ def test_process_delayed_stdio__paused__stdin_pipe(self):
                 stdin=subprocess.PIPE,
                 stdout=subprocess.PIPE,
                 stderr=subprocess.PIPE,
-                __uvloop_sleep_after_fork=True))
```

Comment:
Is this because dunder breaks Cython 3?

Reply:
it was added to get the test to pass, forgotten failure